shared/modules/selectors/networks.ts

```diff
@@ -117,13 +117,13 @@ export const getNetworkConfigurationsByCaipChainId = ({
     ([caipChainId, networkConfig]) => {
       const matchesAccount = Object.values(internalAccounts.accounts).some(
         (account) => {
-          const matchesScope = account.scopes.some((scope) => {
+          const matchesScope = account.scopes?.some((scope) => {
```

Comment:
This is a temp fix as its uncovered that migration 105.ts removes the scope property 😭 

I'll make a short followup to ensure that this scope property is not lost.